src/packages/v4/hooks/useV4TotalTokenSupply.ts

```diff
@@ -0,0 +1,21 @@
+import { WeiPerEther } from '@ethersproject/constants';
+import { useJBContractContext, useReadJbControllerTotalTokenSupplyWithReservedTokensOf } from 'juice-sdk-react';
+import { useMemo } from 'react';
+
+export const useV4TotalTokenSupply = () => {
```

Comment:
How are you feeling about this approach to prefix everything with v4? Idk if its totally necessary given that all the v4 code is housed under `packages/v4`. No big deal, just a thought.

Reply:
Yep fair call, I'll move away from it.